packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -634,6 +636,57 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupExtensions(app, options) {
+        const extensions = options.mobify?.app?.extensions || []
+        app.__extensions = extensions || []
+
+        let _require
+
+        extensions.forEach((extension) => {
+            const setupServerFilePath = path.join(
+                options.buildDir,
+                'extensions',
+                extension,
+                'setup-server.js'
+            )
+
+            // Only eval when there are extensions
+            // this makes it slightly faster for projects that
+            // have no extensions
+            if (!_require) {
+                _require = eval('require')
+            }
+
+            let setupServer
+
+            try {
+                setupServer = _require(setupServerFilePath)
+            } catch (e) {
+                if (e.message && e.message.startsWith('Cannot find module')) {
+                    // skip extensions that don't have a setup-server.js file
+                    return
+                }
+
+                console.error(`Error loading extension ${extension}:`, e)
```

Comment:
I guess we don't have access to the new logger or do we? Might make sense to merge in develop if it is easy and use the logger.

Reply:
I'd like to do that in another PR if you are okay with it